src/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.boxes {
+  flex: 1 0 33%;
+  height: 300px;
+  min-width: 300px;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+}
+
+.box1 {
+  background-color: red;
+}
+
+.box2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.box3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.box4 {
+  background-color: rgb(102, 0, 0);
+  order: 6;
+}
+
+.box5 {
+  background-color: rgb(51, 0, 0);
+  order: 5;
+}
+
+.box6 {
+  background-color: black;
+  order: 4;
+}
```

Comment:
better to use default orders without media

Reply:
without orders I can`t reach the start position of snake laying - 123654